app/views/layouts/application.html.erb

```diff
@@ -1,27 +1,24 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Acebook</title>
-    <%= csrf_meta_tags %>
+<head>
```

Comment:
There are a lot of changes in this file — for instance the `title` tag gets removed. Can we change it so the changes are just exactly what you want?

Reply:
not sure how this happened, wasn't intending to change anything in here. have reverted back to master